src/ParallelAlgorithms/Events/Factory.hpp

```diff
@@ -15,11 +15,11 @@
 namespace dg::Events {
 template <size_t VolumeDim, typename TimeTag, typename Fields,
           typename NonTensorComputeTagsList, typename ArraySectionIdTag = void>
-using field_observations = tmpl::flatten<
-    tmpl::list<ObserveFields<VolumeDim, TimeTag, Fields,
```

Comment:
This appears to only be a formatting change. Can you remove this or put it in a separate commit?

Reply:
Yes, just a remnant of the first method I tried to implement this... I squashed the change, as it doesn't do anything.